Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -170,6 +170,10 @@ class OWTestLearners(OWWidget):
     TARGET_AVERAGE = "(Average over classes)"
     class_selection = settings.ContextSetting(TARGET_AVERAGE)
 
+    shown_scores = \
+        settings.Setting({"AUC", "CA", "F1", "Precision", "Recall",
```

Comment:
**If** list can be stored in json and **if** we care(d) I can replace the set with a list. No big changes in the code, except that conceptually this is a set, not a list.

Can it, do we, should I?